pro/src/routes/Offers/adapters/getFilteredOffersAdapter.ts

```diff
@@ -0,0 +1,116 @@
+import { api } from 'api/v1/api'
+import {
+  ListOffersOfferResponseModel,
+  ListOffersResponseModel,
+} from 'api/v1/gen'
+import { DEFAULT_SEARCH_FILTERS } from 'core/Offers/constants'
+import { Offer, SearchFilters, SearchFiltersAPI } from 'core/Offers/types'
+
+type IPayload = {
+  offers: Offer[]
+}
+
+type GetFilteredOffersAdapter = Adapter<SearchFilters, IPayload, IPayload>
+
+const FAILING_RESPONSE: AdapterFailure<IPayload> = {
+  isOk: false,
+  message: 'Nous avons rencontré un problème lors du chargemement des données',
+  payload: {
+    offers: [],
+  },
+}
+
+const serializeVenue = (venue: ListOffersOfferResponseModel['venue']) => ({
```

Comment:
Je me demande si c'est overkill ou pas : 
```suggestion
import { IVenue /* ... */ } from 'core/Offer/types'
// ...
const serializeVenue = (venue: ListOffersOfferResponseModel['venue']): IVenue => ({
```